CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # CHANGELOG
 
+## 1.0.0-beta.27
+
+### Bug Fixes
+
+Removes the items-center on megabuttons, fixing the alignment issue.
+
+## 1.0.0-beta.26
+
+### Bug Fixes
+
+Updates the megabutton text color to match the figma.
```

Comment:
nit: Let's use capitalization matching the component name, so `MegaButton` instead of `megabutton`.  

nit: Let's rephrase "the figma" to something that will be more relevant to other teams that are sharing this library with us.  They may not know what "figma" we're referring to.  Maybe instead we can call it "the updates to the Design System".